WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -544,6 +545,10 @@ private async Task HandleCoinJoinFinalizationAsync(CoinJoinTracker finishedCoinJ
 			// The fix is already done but the clients have to upgrade.
 			wallet.LogInfo($"{nameof(CoinJoinClient)} failed with exception: '{e}'");
 		}
+		catch (WabiSabiProtocolException wpe) when (wpe.ErrorCode == WabiSabiProtocolErrorCode.WrongPhase)
+		{
+			wallet.LogInfo($"{nameof(CoinJoinClient)} failed because of abnormal round phase change with exception: '{wpe}'");
```

Comment:
Why abnormal? Can this happen in the case of load balancing?

Reply:
If the goal here is to make the message pretty, do not write out `wpe` with the call stack either.

Reply:
However the round number important so `wpe.message`?

Reply:
BTW looking at the title of the issue "in case of detected double-spend". It is not possible to know on client side why was the round ended by the coordinator - the goal is not clear or impossible.

Reply:
Thx for explaining.